tests/test_sonify.py

```diff
@@ -46,6 +47,7 @@ def test_note_hz():
     assert len(y) == 8000 * 2
 
 
+@pytest.mark.xfail
```

Comment:
This one is failing due to a bug in the `_const_interpolator` mode of mir_eval.sonify :facepalm: I'll fix it and get a 0.8.2 release going shortly...

Reply:
Fix is in https://github.com/mir-evaluation/mir_eval/pull/418

Reply:
should i wait for the release and then test again as the last sanity check?

Reply:
0.8.2 is up on pypi now; conda-forge is building and should be up within the hour.

Reply:
cool, everything works now! resolving this.